src/EventTickets/resources/admin/components/EventDetailsPage/TicketTypesSection/TicketTypesRowActions.tsx

```diff
@@ -20,29 +20,30 @@ export function TicketTypesRowActions({tickets, openEditModal}) {
             });
         };
 
-        const deleteItem = async (selected) =>
-            eventTicketsApi.fetchWithArgs('/event/ticket-type/' + ticket.id, {}, 'DELETE');
+        const deleteItem = async (selected) => eventTicketsApi.fetchWithArgs('/ticket-type/' + ticket.id, {}, 'DELETE');
 
         const confirmModal = async () => {
             const confirmDelete = confirm(sprintf(__('Really delete ticket #%d?', 'give'), ticket.id));
 
             if (confirmDelete) {
                 if (await deleteItem(ticket.id)) {
-                    // Refresh data
+                    setTickets((tickets) => tickets.filter((t) => t.id !== ticket.id));
                 }
             }
         };
 
         return (
             <>
                 <RowAction onClick={handleEditClick} displayText={__('Edit', 'give')} />
-                <RowAction
-                    onClick={confirmModal}
-                    actionId={ticket.id}
-                    displayText={__('Delete', 'give')}
-                    hiddenText={ticket.title}
-                    highlight
-                />
+                {ticket.salesCount === 0 && (
```

Comment:
@jdghinson same here. Should we disable the "delete" button and add a tooltip? I'm concerned that simply removing the button could be confusing.

Reply:
Just like in events. I reverted that back and implemented a native alert. https://github.com/impress-org/givewp/pull/7293/commits/62f8b3f9fd0d99edd89381bc16cfc468441dd196